cfme/tests/v2v/test_conversion_host.py

```diff
@@ -0,0 +1,50 @@
+import fauxfactory
+import pytest
+
+from cfme.cloud.provider.openstack import OpenStackProvider
+from cfme.infrastructure.provider.virtualcenter import VMwareProvider
+from cfme.markers.env_markers.provider import ONE_PER_VERSION
+
+pytestmark = [
+    pytest.mark.provider(
+        classes=[OpenStackProvider],
```

Comment:
Question: Can you just do `classes=[OpenStackProvider, VMwareProvider]` here? and get rid of the two `pytest.mark.provider()` statements?

Reply:
I think it is required to be done that way because `fixture_name` is different for both.

Reply:
Cannot do as Vmware needs to be marked as source_provider fixture